### PR TITLE
more info in log

### PR DESCRIPTION
**Pull request by olgavrou**
Regarding #615 adding more info in log

### CODE REVIEW

File: src/host/dns.h [hunk 1]
```diff
@@ -56,7 +56,11 @@ namespace asynchost
              service.c_str(),
              &hints)) < 0)
         {
-          LOG_FAIL_FMT("uv_getaddrinfo failed: {}", uv_strerror(rc));
+          LOG_FAIL_FMT(
```
Comment (jumaffre) [inline src/host/dns.h]: Should we add the same info for the async version a few lines up?

Comment (olgavrou) [reply src/host/dns.h]: yes thanks

**Comment by cimetrics** [comment]
images

**Comment by cimetrics** [comment]
images

**Pull request merged**